app/controllers/trips_controller.rb

```diff
@@ -0,0 +1,21 @@
+class TripsController < ApplicationController
+  def index
```

Comment:
Maybe add a feature spec that an admin can view all the trips?

Also two things to consider for the future:
1. you will probably want to distinguish between future trips and past trips. Or else there will be too much noise on the page to find relevant trips
2. you also probably want to order the trip by upcoming start_date
(maybe add a couple more tickets for these stories in Trello?)

Reply:
I want to do another PR for the index that I added here just for testing purpose, so I will add all the test in the next one. :)

I really love the idea of sorting groups for past/future, thanks! I will add Trello cards for it!

Reply:
If the action is added, it should have a spec with it (the idea is that no code is added without tests)